package.json

```
@@ -100,6 +100,11 @@
       }
     ],
     "commands": [
+      {
+        "command": "one.metadata.showMetadata",
+        "title": "hello test",
```

```suggestion
        "title": "View Detailed Information",
```

For users, it will be better to use the term 'information' or 'details' instead of 'meta'